CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 # Changelog for coreSNTP Library
+
+## v1.1.1 (August 2022)
```

Comment:
Is the date correct?

Reply:
So it was at time of putting the review up, leaving it as August would also keep it consistent with every other library that was version bumped.